src/basis/data.js

```diff
@@ -2436,14 +2438,18 @@
   // Accumulate dataset changes
   //
 
-  Dataset.setAccumulateState = (function(){
+  (function(){
+    var PREVENT_ACCUMULATIONS = {};
     var proto = ReadOnlyDataset.prototype;
     var eventCache = {};
     var setStateCount = 0;
     var urgentTimer;
     var realEvent;
 
     function flushCache(cache){
+      if (cache === PREVENT_ACCUMULATIONS)
```

Comment:
Because if this line was reached, then this dataset was already flushed and prevented